src/cljc/orcpub/dnd/e5/spells.cljc

```diff
@@ -1133,15 +1133,17 @@ While on the Ethereal Plane, you can see and hear the plane you originated from,
     :description "Your body becomes blurred, shifting and wavering to all who can see you. For the duration, any creature has disadvantage on attack rolls against you. An attacker is immune to this effect if it doesn't rely on sight, as with blindsight, or can see through illusions, as with truesight."
     }
    
-   {:name "Booming Blade"
+   #_{:name "Booming Blade"
```

Comment:
This comments out this block, removing the spell from the list.
Is this intentional? There are several spells commented out already in this file.

Reply:
Yes, its not in SDR
https://github.com/larrychristensen/orcpub/issues/192

Reply:
Makes sense.
I'll merge.